pkg/drain/nodegroup.go

```diff
@@ -235,3 +279,12 @@ func cordonStatus(desired bool) string {
 	}
 	return "uncordon"
 }
+
+func isEvictionErrorRecoverable(err error) bool {
```

Comment:
We have a string contains in here somewhere if you want something like this.

But I think we might be able to detect the error a bit better perhaps than relying on the text.

Reply:
This has been replaced with more intelligent error checking logic.